.gitlab-ci.yml

```diff
@@ -0,0 +1,25 @@
+image: spritecloud/ruby-selenium-runner:2.4
```

Comment:
Can we remove the gitlab-ci and related calliope keys in the config files? We can add the functionality to run and upload the test(results) on our gitlab environment.

Reply:
Indeed, removed gitlab-ci file